pfcpiface/p4rtc.go

```diff
@@ -0,0 +1,885 @@
+package main
```

Comment:
If this is equivalent of bess_pb, we should make a separate package maybe, instead of main?